static/js/util/profile_edit.js

```diff
@@ -7,8 +7,8 @@ import Checkbox from 'material-ui/Checkbox';
 import Geosuggest from 'react-geosuggest';
 import R from 'ramda';
 import ReactTelInput from 'react-telephone-input';
-
 import DateField from '../components/inputs/DateField';
```

Comment:
we normally leave one blank line between outside dependencies and our own modules in the imports at the top of a JS file (sorry, this stuff isn't written down anywhere)